examples/01-Modeling-Setup/Configurations.py

```diff
@@ -108,7 +115,7 @@
 # ~~~~~~~~~~~~~~~~~~~~
 # This section we create a new Icepak project and iport the step.
 
-app = Icepak(projectname="new_proj_Ipk")
+app = Icepak(projectname="new_proj_Ipk", non_graphical=False)
```

Comment:
Is it expected to have Icepak always run in non-graphical  even when not in the documentation process/

Reply:
effectively that bool is ignored 2nd time because Desktop is already opened. Anyway good catch I've removed it